oak_debug_service/src/lib.rs

```diff
@@ -0,0 +1,99 @@
+//
+// Copyright 2024 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+pub mod proto {
+    pub mod oak {
+        pub mod debug {
+            tonic::include_proto!("oak.debug");
+        }
+    }
+    pub mod perftools {
+        pub mod profiles {
+            tonic::include_proto!("perftools.profiles");
+        }
+    }
+}
+use std::time::Duration;
+
+use pprof::protos::Message as _;
+use prost::Message as _;
+use proto::{
+    oak::debug::{debug_service_server::DebugServiceServer, CpuProfileRequest, CpuProfileResponse},
+    perftools::profiles::Profile,
+};
+
+use crate::proto::oak::debug::debug_service_server::DebugService;
+
+const SAMPLE_FREQUENCY: i32 = 100;
```

Comment:
Is this a percentage? Maybe call it SAMPLE_FREQUENCY_PERCENT

Reply:
No, it's frequency, in Hz (100 interrupts per second).

Reply:
Even more reason for adding the unit to the name :)